public/javascripts/items.js

```diff
@@ -7,28 +7,105 @@ const fetchParams = {
   credentials: 'include'
 }
 
-const params = completed =>
-  Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
+const params = data =>
+  Object.assign( {}, FETCH_PARAMS, { body: JSON.stringify( data ) } )
 
-$(document).ready( () => {
-  $( '.title' ).click( event => {
-    const element = $( event.target )
-    const id = element.data( 'id' )
-    const completed = ! element.data( 'completed' )
+const RETURN_KEY = 13
```

Comment:
I'd move this up to be next to `FETCH_PARAMS`, just to create some visual distinction between `CONSTANTS` and module functions (like `params`)